### PR TITLE
Include all packages in search

### PR DESCRIPTION
**Pull request by moxw**
Fixes: #1056 

fix(DetailsSearch.vue): modify condition to push item based on childLeafs length
fix(findings.spec.js): add e2e test for actionview package search in AppMap

### CODE REVIEW

**Comment by ahtrotta** [comment]
Hi @moxw, thanks for this PR. Can you fix the PR so that the build passes before I review this?

File: packages/components/tests/e2e/specs/appmap/viewFilter.spec.js [hunk 1]
```diff
@@ -125,10 +125,10 @@ context('AppMap view filter', () => {
       cy.get('.trace .trace-node').should('have.length', 5);
     });
 
-    it('hides unlabeled code', () => {
+    it.only('hides unlabeled code', () => {
```
Comment (ahtrotta) [inline packages/components/tests/e2e/specs/appmap/viewFilter.spec.js:128]: Hi @moxw can you please remove the `.only` call?

Comment (moxw) [reply packages/components/tests/e2e/specs/appmap/viewFilter.spec.js:128]: Removed.

Comment (ahtrotta) [reply packages/components/tests/e2e/specs/appmap/viewFilter.spec.js:128]: I still see it

Comment (moxw) [reply packages/components/tests/e2e/specs/appmap/viewFilter.spec.js]: Apologies, was pushing it from wrong directory, should be fixed now.

**Review by ahtrotta — APPROVED** [review]
Looks good, thanks!

**Pull request merged**